pkg/validate/serialize.go

```diff
@@ -17,28 +33,184 @@ func Verify(yamlFileName string) error {
 	}
 
 	// Value returned is a marshaled go type (CSV Struct).
-	_, err = unmarshal(rawYaml)
+	csv, err := unmarshal(rawYaml)
 	if err != nil {
 		return fmt.Errorf("Error unmarshalling YAML to OLM's csv type for %s file:  #%s ", yamlFileName, err)
 	}
 
-	fmt.Printf("%s is verified", yamlFileName)
-	return nil
+	// Contains error logs for all missing optional and mandatory fields.
+	errorLog := csvInspect(csv)
+	checkManifestResult(errorLog.warnings)
+	checkManifestResult(errorLog.errors)
```

Comment:
can we put this in the if statement below?

Reply:
Do you want to check for `nil` before calling `checkManifestResult`?

Reply:
I just think since you are printing the errors only if it's not nil, and you are raising an error only if its not nil, might as well include all of that under the same IF no?

Reply:
Done.